File: Task/GUI-Maximum-window-dimensions/Visual-Basic-.NET/gui-maximum-window-dimensions-1.vb
Imports System.Drawing
Imports System.Windows.Forms

Module Program
    Sub Main()
        Dim bounds As Rectangle = Screen.PrimaryScreen.Bounds
        Console.WriteLine($"Primary screen bounds:  {bounds.Width}x{bounds.Height}")

        Dim workingArea As Rectangle = Screen.PrimaryScreen.WorkingArea
        Console.WriteLine($"Primary screen working area:  {workingArea.Width}x{workingArea.Height}")
    End Sub
End Module

File: Task/GUI-Maximum-window-dimensions/C-sharp/gui-maximum-window-dimensions-1.cs
using System;
using System.Drawing;
using System.Windows.Forms;

static class Program
{
    static void Main()
    {
        Rectangle bounds = Screen.PrimaryScreen.Bounds;
        Console.WriteLine($"Primary screen bounds:  {bounds.Width}x{bounds.Height}");

        Rectangle workingArea = Screen.PrimaryScreen.WorkingArea;
        Console.WriteLine($"Primary screen working area:  {workingArea.Width}x{workingArea.Height}");
    }
}
